app/scripts/components/common/datepicker/index.scss

```diff
@@ -38,6 +38,16 @@
         background-color: utils.color('gray-10') !important;
       }
     }
+
+    &--now:disabled {
```

Comment:
I can't see the button getting disabled. (Therefore, this style isn't applied.) Am I missing anytyhing? 
 
<img width="1884" alt="Screenshot 2024-08-09 at 11 18 06 AM" src="https://github.com/user-attachments/assets/08f76133-90e0-49be-8107-066873472e0f">

Reply:
I might be misunderstanding your question, so let's catch up over a call if I do. But I think this is an edge case, which is difficult to reproduce now when we added a limit to the calendar min and max dates.

Here's a simple scenario: you have a dataset with temporal extent from Aug 5, 2024- Aug 10, 2024. If you open the calendar picker, you would see the `now` date (e.g Aug 12 which is today) and it will be disabled due to the temporal extent of the dataset. It will also be colored yellow due to `react-calendar` and our style aims to override that (to align with the USWDS behavior).

Reply:
Ah I see ! I somehow thought we want to disable 'now' (marked with a yellow background) totally. Thanks for walking through the case 🙇